core/Property/Property.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Property;
+
+use Exception;
+use Piwik\Site;
+
+/**
+ * Provides access to individual properties.
+ */
+class Property extends Site
+{
+
+    public function getSettingValue($name)
```

Comment:
I currently do not plan to make this method available in `Site` but can do if needed later.